_examples/blockchain.adoc

```diff
@@ -0,0 +1,107 @@
+:page-project_links: ["https://github.com/Jiri-Kremser/bitcoin-insights"]
+
+[[introduction]]
+== Introduction
+
+Cryptocurrencies attract various groups of people. Among them could be
+investors, people from retail, tech enthusiasts, crypto-anarchists, etc.
+We are not going to focus on anything other than the raw technology behind the
+Blockchain, leaving aside all the ideology and hype that comes with the Bitcoin.
+
+The https://spark.apache.org/graphx/[GraphX] based notebook uses this https://github.com/spark-notebook/spark-notebook[spark-notebook] as the notebook technology,
+while the https://graphframes.github.io/[GraphFrames] based notebook uses project Jupyter. Also the second
+notebook connects to an existing spark cluster that was created by Oshinko tools.
+
+[[architecture]]
+== Architecture
+
+A Jupyter/Spark notebook is an interactive compute environment that interleaves
+documentation, code examples, and graphical output.
+
+Similar to our other notebook examples, these notebooks are the Spark drivers that interactively
+send the commands to the Spark cluster for the evaluation. They can either spawn
+their own standalone Spark or connect to an external one.
+
+[[installation]]
+== Install and run the notebook
+
+Installing our notebook is very simple:  we just need to create an OpenShift project and install and run the notebook application.
+Make sure you're logged in to OpenShift and have selected the right project, and then execute the following two commands:
+
+....
+oc new-app jkremser/bitcoin-notebook:tutorial-1.0.0
+oc expose svc/bitcoin-notebook
+....
+
+for the GraphFrames notebook and the following for the GraphX notebook:
+
+....
+oc new-app jkremser/bitcoin-spark-notebook:tutorial-1.0.0
+oc expose svc/bitcoin-spark-notebook
+....
+
+Once both routes are exposed, the notebooks should be available. You
+can find the URLs if you run `oc get routes`.
+It is also possible for the Jupyter notebook to set
+environment variable for connecting to an external Spark cluster.
+
+To do that, you can optionally run this:
+`oc new-app jkremser/bitcoin-notebook:tutorial-1.0.0 -e SPARK_MASTER=spark://c:7077`
+
+Where `c` is the hostname of the Spark cluster.
+
+[[usage]]
+== Usage
+Both notebooks come with some prepared data in the parquet files, but part of the
+project is also the https://github.com/Jiri-Kremser/bitcoin-insights/tree/master/parquet-converter[parquet-converter]
+that takes the raw Blockchain data and converts
+it to the format that is digestible by Spark. It assumes the official bitcoin wallet to be installed
+that itself downloads the whole Blockchain (~150 GB). However, it works also with
+a partially downloaded Blockchain.
+
+* Jupyter Notebook (GraphFrames and NetworkX)
++
```

Comment:
i think you should add something here about the 2 pynb's included, i was a little confused when i saw "blockchain.ipynb" and "graphframes.ipynb". it would be nice to give the user a little advice, like "start with the blockchain nb then move on to graphframe" or similar.

Reply:
Mike, I removed the `graphframes.ipynb` from the image (https://github.com/Jiri-Kremser/bitcoin-insights/commit/efe5821a55736a2de00d3a562ca736126bd326dc) and add couple of sentences in https://github.com/radanalyticsio/radanalyticsio.github.io/pull/130/commits/72ad0b5455542d43d36eecbeae4d0de742b7bdb8